app/views/qa/show.html.erb

```diff
@@ -33,14 +33,14 @@
         <% elsif multiple_question? %>
           <%= render "govuk_publishing_components/components/checkboxes", {
             name: "#{current_facet['facet']['key']}[]",
-            heading: "Select all that apply",
+            heading: current_facet["question"],
```

Comment:
So this is the same as the title of the page? i.e. What's in the <H1>?

That looks fine to me, but I wonder if someone like @vanitabarrett or @andysellick could review.

Reply:
Have you considered only having the legend (perhaps setting it to 'Select all that apply'), and removing the hint text as in this example? https://govuk-publishing-components.herokuapp.com/component-guide/checkboxes/without_hint_text

Reply:
I don't think that works for us, it would change the design.

Reply:
That would also introduce an a11y issue (as per Anika's comment on Slack). Documenting on GitHub:

> "Select all that apply" is not an adequate legend. The legend is there so that screen reader users can understand what a checkbox is about when they read it out of context. Screen reader users often just navigate from one form control to the next, so are likely to miss the question. It's the question that should be the legend.

Reply:
It looks like "Select all that apply" is the [default hint text](https://govuk-publishing-components.herokuapp.com/component-guide/checkboxes/with_legend_as_page_heading).

So perhaps, rather than `visually_hide_heading`, we can use `is_page_heading: true` instead?